### DISCUSSION TITLE
list of events with remaining seats

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#118] [Q&A] [answered]
Hi Franky,
I would like the list of events, only those for which there is at least one place left to book. I couldn't find a parameter to the shortcode [eme_events] to do that. Is there a simple other way or do i need to code?

**Comment by liedekef** [comment]
There is the EME option to hide RSVP-full events. If you want to just sometimes do it, use an list-template containing [eme_if] that evaluates the number of free seats.

**Comment by f14m07** [comment]
I put that on a page `[eme_if tag='#ESC_AVAILABLESEATS' ge='1'][eme_events][/eme_if]`
but no events were listed

**Reply by liedekef** [reply]
this doesn't work because #_AVAILABLESEATS is evaluated per event, so it can't do anything here. You need to use it in a template per event.

**Comment by f14m07** [comment]
in fact it works by putting on the page `[eme_events template_id=77]`
and in the template_id 77 I put 
`
[eme_if tag='#ESC_AVAILABLESEATS' notcontains='0']
<table style="width: 100%;">
<tbody>
<tr>
<td style="width: 25%;">#_LOCATIONIMAGETHUMB</td>
<td style="padding: 8px;">Nom de la sortie <strong>#_LINKEDNAME </strong>
du <span style="color: blue;">#_STARTDATE #_STARTTIME au #_ENDDATE #_ENDTIME</span>
voilier <b>#_LOCATIONNAME</b> chef de bord <b>#_AUTHORDISPNAME</b>
il reste <span style="color: red;">#_AVAILABLESEATS </span> place(s) disponible(s)
#_APPROVEDSEATS place(s) est (sont) réservée(s) [eme_if tag='#_IS_USER_MEMBER_OF{Equipiers}' value='1'] par #_ATTENDEES [/eme_if]
Participation #_EVENTPRICE</td>
</tr>
</tbody>
</table>
&nbsp;
but without 
`[/eme_if] `

**Comment by liedekef** [comment, marked as answer]
because you have another eme_if in there. If you're going to nest eme_if statements, use eme_if2, eme_if3 etc ...

**Comment by f14m07** [comment]
thanks Franky, that's fine
